server/syntax_gen.js

```diff
@@ -0,0 +1,77 @@
+module.exports = function () {
+	var fs = require('fs');
+	var path = require('path');
+	var nodes = require('./nodetypes/nodetypes.js');
+
+	var file_path = path.join(__dirname, '../client/libs/ace-builds/src-min/snippets/fbp.js');
+
+	this.generateNodetypes = function() {
+		var names = nodes.getNodeTypesName();
```

Comment:
Our CI got an error because It could not find the nodetypes descriptions.

If this happens, we should not kill the server. Also, we could let the autocomplete to run only for created variable.

So... if we detect an error to get the Nodetypes try going for a default configuration and warn it in the console.log of the server. Something like ```Warning: Could not find the nodetypes descriptions```